Dockerfile

```diff
@@ -4,6 +4,11 @@ ADD . /usr/src/sriov-cni
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
+
+RUN apk add --update --virtual build-dependencies build-base linux-headers && \
+    cd /usr/src/sriov-cni && \
+    make clean && \
+    make build
 RUN cd /usr/src/sriov-cni && make build
```

Comment:
Remove this line?

Reply:
Good catch! forgot to remove this line :)